docs/setup-and-usage.md

```diff
@@ -127,7 +127,7 @@ wget https://raw.githubusercontent.com/tellor-io/telliot/master/configs/config.j
 ### Create .env file
 
 Most commands require some secrets and these are kept in a separate `configs/.env`. This is a precaution so that are not accidentally exposed as part of the main config. Make a copy of the `env.example` and edit with your secrets.
-For a working setup it is required to at least add one private key in your `"ETH_PRIVATE_KEYS"` environment variable. all of public addresses can be determined from your private keys.
+For a working setup it is required to at least add one private key in your `"ETH_PRIVATE_KEYS"` environment variable. all of public addresses can be determined from your private keys. Also you may need variables in the format of `$PSR$_KEY` for getting a specific indexes.json api \(required if you use authenticated API's\).
```

Comment:
I think the text should be more like: `Any env variable from the .env file is substituted in the API URL. For example......`